example/java/build_library.sh

```diff
@@ -6,8 +6,13 @@ mkdir dist
 
 # build native library
 ./prepare.sh
-cp build/shared_model/bindings/libirohajava.jnilib dist/libirohajava.jnilib
 
+unamestr=`uname`
+if [[ "$unamestr" == 'Linux' ]]; then
+    cp build/shared_model/bindings/libirohajava.so dist/libirohajava.so
+elif [[ "$unamestr" == 'Darwin' ]]; then
+    cp build/shared_model/bindings/libirohajava.jnilib dist/libirohajava.jnilib
+fi
```

Comment:
else -> exit?

Reply:
I guess no. At least you will have jar files.